### ISSUE TITLE
Loading Saved queries isn't working properly with new Polymer desktop app

### ISSUE THREAD

**Issue by jaydude5** [jarrodek/ChromeRestClient#592]
I have a bunch of queries I saved previously before the app was updated to be a desktop app. When I load a new query while I'm in the app, it brings up the title of the new saved query but leaves the URL and all the query parameters populated with whatever query I was looking at before. I have to choose a saved query, then close the app, then re-open it to actually get the right URL/parameters.

## Expected outcome
When I change queries during a session, it should load the URL and parameters for my new saved query. There is even an animation suggesting it's loading the new saved data.

## Actual outcome
When I switch queries via the Saved menu, it changes the title of my query, but not the host/path/parameters

## Versions
App: 7.7.26.168-stable
Chrome: 49.0.2623.112
Platform: Mac OS X (10_10_5)

## Steps to reproduce
1. Save more than 1 query in the Saved list (I didn't save them to Google Drive and this might only occur with queries that were saved before the app was updated)
2. Open the app and load one query
3. Close the app and reopen
4. With the first saved query loaded, go to the Saved menu again and select a different saved query
5. Observe that only the title changes but none of the parameters or URL updates to reflect the newly selected saved query.

**Comment by jarrodek** [comment]
Hi,

Thank you for your issue report.
Is this happening when you have detailed view opened?

Thanks.

**Comment by jaydude5** [comment]
Yes,

When I first open the app, it loads the URL and params for a saved query (it starts me on the "Request" menu item in the left sidebar). 

Then, I choose "Saved" in the left sidebar, choose a different query from the original one, and click "OPEN" on the right side of the "Saved" window.

This takes me back to the "Request" view, and the title at the top reflects the new Saved item I chose, but the URL and params still show the same ones as the first request that loaded when I opened the app.
